src/main/java/backend/UpdateController.java

```diff
@@ -87,7 +89,13 @@ private void openRepositoriesInFilters(List<FilterExpression> filterExprs) {
         return filterExprs.stream()
                 .filter(Qualifier::hasUpdatedQualifier)
                 .flatMap(filterExpr -> allModelIssues.stream()
-                        .filter(issue -> Qualifier.process(models, filterExpr, issue)))
+                        .filter(issue -> {
+                            try {
+                                return Qualifier.process(models, filterExpr, issue);
+                            } catch (SemanticException e) {
+                                return false;
+                            }
+                        }))
```

Comment:
Yup, that's fine. We'll just do something simple here, pending something more complex after the repo picker/multiple errors and warnings.